Community standard documents (#41)
kolibrios-nextgen/kolibrios-nextgen: CONTRIBUTING.md

# Contributing guide

__Contributing__ guide of __KolibriOS-NG__ project

## Overview

The process of contributing to the project is described in detail in this document

## Code of Conduct

The first thing to do is to read and follow our [Code of Conduct](./CODE_OF_CONDUCT.md)

## Type of contributing

There are two main types of contributions: submitting issues about problems and submitting pull requests. Each of these types of contributions is described in detail below

## Issues

> You can help us by submitting issues to repository. At the moment there are 2 main forms of submitting an issue in the project: _bug reports_ and _feature requests_. All existing forms are provided with brief explanations

Reviewer: ```suggestion
You can help us by submitting issues to repository. At the moment there are 2 main ways of submitting an issue in the project: _bug reports_ and _feature requests_. All existing forms are provided with brief explanations
```

> - Reports are suitable if you find a __bug__ in the some part of the project and want to create a report

Reviewer: ```suggestion
- Reports are suitable if you find a __bug__ in some part of the project and want to create a report
```

- Also there is a possibility submit a __feature request__ if it seems to you that the project lacks any feature

## Pull requests

You can also help us by submitting pull requests. The process of submitting a pull request consists of several steps:

- Find what you want to implement or improve
- Make a fork
- Create a branch with name in the appropriate style
- Make and test the changes
- Create commits according to the accepted style

> - Create and submit pull request

Reviewer: ```suggestion
- Create and submit a pull request
```

> - Waiting for passing CI/CD pipelines and code review

Reviewer: ```suggestion
- Wait for CI/CD pipelines and code review to pass 
```

> After submitting a pull requests at least two project participants must conduct a code review, after which the changes are corrected (if it's necessary) and merged into the project

Reviewer: ```suggestion
When a pull request is submitted, at least two project participants must conduct a code review, after which the changes are corrected (if it's necessary) and merged into the project
```
